test/test_sampler_sim_workspace.py

```diff
@@ -10,16 +10,18 @@
 from synth_simulation_workspace import SynthSimulationWorkspace
```

Comment:
We should make tests composable and test over a matrix:
(several different simulations) x (different pools) x (different storage)